terraform/slurm_cluster/main.tf

```diff
@@ -133,6 +133,10 @@ module "slurm_files" {
   install_dir             = lookup(var.controller_hybrid_config, "install_dir", null)
   munge_mount             = lookup(var.controller_hybrid_config, "munge_mount", null)
 
+  # Providers
```

Comment:
`slurm_cluster` module is not used, let's not make any changes to it.

Reply:
Done.